cf-deployment.yml

```diff
@@ -305,6 +305,7 @@ instance_groups:
             override: true
             refresh-token-validity: 2592000
             scope: cloud_controller.read,cloud_controller.write,openid,password.write,cloud_controller.admin,scim.read,scim.write,doppler.firehose,uaa.user,routing.router_groups.read,routing.router_groups.write
+            secret: ''
```

Comment:
I'm curious about this line. One of the goals of cf-deployment is to remove as much unneeded configuration from the manifest as possible. Any time we provide an empty string in the manifest, it seems like an opportunity to ask about default values provided by the release.

Can you explain why we need to explicitly set this to an empty string? What is the default value if you don't set this value?